src/components/views/dialogs/CreateSubspaceDialog.tsx

```diff
@@ -41,9 +41,9 @@ const CreateSubspaceDialog: React.FC<IProps> = ({ space, onAddExistingSpaceClick
 
     const [busy, setBusy] = useState<boolean>(false);
     const [name, setName] = useState("");
-    const spaceNameField = useRef<Field>();
+    const spaceNameField = useRef() as RefObject<Field>;
```

Comment:
Why do we need to cast it ? An issue between `MutableRefObject` and `RefObject` ?

Reply:
Indeed, plus we don't really want to encourage manual mutation, it should be managed entirely by React and only accessed by us. But yeah without it React + Typescript don't fit the type properly

![image](https://user-images.githubusercontent.com/2403652/231543235-70912243-1141-4587-b40d-dc45379c3b23.png)